Added an Entity Query snippet (Monobehaviour workflow) (#1138)
spatialos/gdk-for-unity: docs/workflows/monobehaviour/interaction/commands/world-commands.md

  * [`WorldCommands.EntityQuery.Request request`]({{urlRoot}}/api/core/commands/world-commands/entity-query/request): The command request payload.
  * `Action<WorldCommands.EntityQuery.ReceivedResponse> callback`: Optional. A callback that will be called when the command response is received.

> The following code snippet shows an example of how to query for a specific entity based on its ID.

Reviewer: ```suggestion
For example, if you wished to query for a specific entity based on its ID and return all the components on that entity:
```

```csharp
worldCommandSender.SendEntityQueryCommand(

>             new WorldCommands.EntityQuery.Request(

Reviewer 2: please fix the indentation of this snippet

>             new WorldCommands.EntityQuery.Request(

Reviewer 2: please fix the indentation of the code snippet

                new EntityQuery
                {
                    Constraint = new EntityIdConstraint(entityId.Id),
                    ResultType = new SnapshotResultType()
                }),
            Callback);
```

> Where `Callback` have the following signature:

Reviewer 2: ```suggestion
Where `Callback` has the following signature:
```
